src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml.cs

```diff
@@ -187,15 +189,9 @@ private void ShowHideNodeAutoCompleteControl(ShowHideFlags flag)
             ShowHidePopup(flag, NodeAutoCompleteSearchBar);
         }
 
-        private void ShowDNAAutocompleteBar()
+        private void ShowNodeAutoCompleteBar(PortViewModel viewModel)
         {
-            if (ViewModel.NodeAutoCompleteSearchViewModel.IsOpen)
-            {
-                return;
-            }
-            var window = new DNAAutocompleteBar(Window.GetWindow(this), ViewModel.NodeAutoCompleteSearchViewModel);
-            window.Show();
-            ViewModel.NodeAutoCompleteSearchViewModel.PortViewModel.SetupPlaceDNAAutocompletePlacement(window);
+            RequesNodeAutoCompleteBar?.Invoke(Window.GetWindow(this), viewModel);
```

Comment:
Would `Window.GetWindow(this)` work in DynamoRevit context?

Reply:
Yes, should be fine. I'll make an integration ahead of time in D4R for testing purposes later this week.